src/components/WebCLI/WebCLI.tsx

```diff
@@ -197,30 +211,34 @@ const WebCLI = ({
     setShouldShowHelp(!shouldShowHelp);
   };
 
-  // If we do not have an address then do not try and render the UI.
-  if (!wsAddress) {
-    return null;
-  }
-
   return (
     <div className="webcli">
       <WebCLIOutput
         content={output}
         showHelp={shouldShowHelp}
         setShouldShowHelp={setShouldShowHelp}
         helpMessage={
-          <>
-            Welcome to the Juju Web CLI - see the{" "}
-            <a
-              href={externalURLs.cliHelp}
-              className="p-link--inverted"
-              rel="noreferrer"
-              target="_blank"
-            >
-              full documentation here
-            </a>
-            .
-          </>
+          // If errors exist, display them instead of the default help message.
+          Object.values(InlineErrors).some((error) => hasInlineError(error)) ? (
```

Comment:
I think we should update `hasInlineError` to check all errors if you don't pass a key.

Reply:
Done.